doc/adding_users.rst

```diff
@@ -74,15 +74,10 @@ Adding users (lab admins)
    ceph-sepia-secrets.git/ansible/inventory/group_vars/all.yml.  Add
    to teh `lab_users` section.  Submit a pull request.
 
-#. Create a user on teuthology.front.sepia.ceph.com::
-
-     adduser --disabled-password USER
-     cd /home/USER
-     mkdir .ssh
-     cat > .ssh/authorized_keys
-     <ssh key>
-     ^D
-     chown -R --reference=. .ssh
-     chmod 700 .ssh
-     chmod 600 .ssh/*
+#. Run ansible to create the user::
 
+     cd <path-to-ceph-cm-ansible>
```

Comment:
We should probably mention the step of adding the user to the secrets repo here somewhere.